carball/json_parser/game.py

```diff
@@ -403,7 +403,8 @@ def parse_replay(self):
                         # ball_is_sleeping = RBState.get('Sleeping', True)
                         data_dict = BallActor.get_data_dict(actor_data, version=self.replay_version)
                         player_ball_data['ball'][frame_number] = data_dict
-                    elif actor_data["TypeName"] == "Archetypes.Ball.Ball_Basketball":
+                    elif actor_data["TypeName"] == "Archetypes.Ball.Ball_Basketball" or \
```

Comment:
alternative is that you could do .lower and then compare to the type .lower
So it is 100% case insensitive

Reply:
but shouldn't we do it at every type then?

Reply:
Yes most likely